common/src/main/java/com/trainlab/service/impl/UserServiceImpl.java

```diff
@@ -83,16 +77,21 @@ public List<User> findAll() {
     public User update(UserRequest userRequest, Long id) {
         User user = findById(id);
         User updated = userMapper.partialUpdateToEntity(userRequest, user);
-        setEncodedPassword(updated);
+        encodePassword(updated);
         return userRepository.saveAndFlush(updated);
     }
 
+    private void encodePassword(User user) {
```

Comment:
```suggestion
    private void encodePasswordAndSetToUser(User user) {
```

Вообще, есть ли смысл выносить в отдельный метод 2 строки кода только потому что они повторяются 2 раза в сервисе?

Reply:
я этим и руководствовался)